recipes-devtools/cdrtools/cdrtools-native_3.01.bbappend

```diff
@@ -0,0 +1 @@
+CVE_VERSION = "${@d.getVar('PV').split('a')[0]}"
```

Comment:
How about following?

```
CVE_VERSION = ${REALPV}
```

Reply:
Thank you for your advice.
I confirmed it works correctly.